integration-tests/tests/api/target/usage.spec.ts

```diff
@@ -2776,36 +2829,41 @@ test.concurrent(
       },
     });
 
-    await waitFor(10000);
+    let firstSchemaCheckId: string | undefined;
+    await pollFor(async () => {
+      try {
+        const used = await token
```

Comment:
I dont love having to run these checks inside the pollFor to determine when something has cycled through the system, but I couldn't figure out a better way for a number of these cases.